osgp/platform/osgp-throttling-service/src/main/java/org/opensmartgridplatform/throttling/PermitsPerNetworkSegment.java

```diff
@@ -134,9 +131,74 @@ public boolean releasePermit(
         this.permitRepository.releasePermit(
             throttlingConfigId, clientId, baseTransceiverStationId, cellId, requestId);
 
+    if (this.useHighPrioPool()) {
+      this.permitReleasedNotifier.notifyPermitReleased(baseTransceiverStationId, cellId);
+    }
+
     return numberOfReleasedPermits == 1;
   }
 
+  private boolean useHighPrioPool() {
```

Comment:
Can't we use an explicite use boolean in the properties here?

Reply:
Added explicit property